xarray/backends/netCDF4_.py

```diff
@@ -244,6 +245,16 @@ def __init__(self, netcdf4_dataset, mode='r', writer=None, opener=None,
     def open(cls, filename, mode='r', format='NETCDF4', group=None,
              writer=None, clobber=True, diskless=False, persist=False,
              autoclose=False):
+        import netCDF4 as nc4
+        from distutils.version import LooseVersion
```

Comment:
You can put this import at the top level. (We only import netCDF4 inside functions/methods to avoid the hard dependency, but disutils is part of Python's standard library.)